build.bash

```diff
@@ -65,16 +60,18 @@ for row in $(cat pages.json | jq --compact-output '.[]'); do
 	file=$(_jq '.file')
 	header=$(_jq '.header')
 
-	dir=$(dirname ${file})
-
-	mkdir -p ${dir}
-  ${install_dir}/lava help ${topic} > ${file}
-  fix_headers ${file}
-  add_main_header ${file} ${header}
+	mkdir -p "$(dirname "${file}")"
+	${install_dir}/lava help "${topic}" > "${file}"
+	check_status $? "error: could not retrieve topic help for topic ${topic}"
+	fix_headers "${file}"
+	add_main_header "${file}" "${header}"
 done
 
-curl -sSL https://github.com/rust-lang/mdBook/releases/download/v0.4.37/mdbook-v0.4.37-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=${install_dir}
-check_status $? "error: could not install mdBook"
+${install_dir}/lava version > "src/last_version.md"
+
+if ! install_mdBook "${install_dir}"; then
+	echo 'error: could not install mdBook' >&2
+	exit 1
+fi
```

Comment:
I would install mdbook just after installing lava. So both calls to `install_` are together and we fail soon if any of the dependencies is not available.